### ISSUE TITLE
Numeric hintchars don't work with the programmer dvorak keyboard layout

### ISSUE THREAD

**Issue by tom-mi** [tridactyl/tridactyl#3257]
Tridactyl version: 1.20.4
Firefox version: Mozilla Firefox 84.0.1
Operating system: linux

*   Brief description of the problem: Numeric hints don't work with the [programmer dvorak](https://www.kaufmann.no/roland/dvorak/) keyboard layout.

*   Steps to reproduce:

   1. Use programmer dvorak keyboard layout (e.g. `setxkbmap us -variant dvp`)
   2. Use hint mode
   3. Enter number characters (on programmer dvorak, you need SHIFT to access them)
   4. nothing happens (using the number keys on the numpad works as expected though)

*   URL of the website the bug happens on: any

*   Config (in a new tab, run `:viewconfig`, copy: the url and paste it somewhere like gist.github.com): https://gist.github.com/tom-mi/c299aa9a990adf6b00e4be48c3e070a4

*   Contents of ~/.tridactylrc or ~/.config/tridactyl/tridactylrc (if they exist):
(relevant part)
```
set hintfiltermode vimperator
set hintnames numeric
```
or 
```
set hintfiltermode vimperator
set hintchars 1234567890
set hintnames short
```
– both don't work.

This might be related to #1775

**Comment by bovine3dom** [comment]
Yeah, this is a longstanding problem - https://github.com/tridactyl/tridactyl/issues/1769#issuecomment-517790495 - but I don't think we have an issue open for it yet.

(AZERTY has a similar requirement where you must press shift to access numerals)

**Comment by bovine3dom** [comment]
`tri.keyseq.canonicaliseMapstr("<CS-6>")` gives `<C-6>` which might be related

**Comment by bovine3dom** [comment]
Fixed in 07fde5ea
